spoonbill/__init__.py

```diff
@@ -76,19 +120,30 @@ class FileFlattener:
     :param xlsx: Generate combined xlsx table
     """
 
-    def __init__(self, workdir, options, tables, root_key="releases", csv=None, xlsx="result.xlsx", language=LOCALE):
+    def __init__(
+        self,
+        workdir,
+        options,
+        tables,
```

Comment:
Hm, `tables`, `multiple_values` and `root_key` (`pkg_type`) are basically part of Analyzer so maybe its better to replace them with just analyzer